docs/docs/api/hooks/input/use-reanimated-focused-input.md

```diff
@@ -23,22 +31,24 @@ The `input` property from this hook is returned as `SharedValue`. The returned d
 
 ```ts
 type FocusedInputLayoutChangedEvent = {
-  target: number; // tag of the focused TextInput
-  layout: { // layout of the focused TextInput
+  // tag of the focused TextInput
```

Comment:
Slightly strange, that the comment was moved above, but comments below were not 😐

Reply:
I'll update to prettier v3 to see if we can keep this behavior (comment on the same line, after an open bracket)

Reply:
I cannot keep it on the same line, so I kept it this way:

```ts
type FocusedInputLayoutChangedEvent = {
  target: number; // tag of the focused TextInput

  // layout of the focused TextInput
  layout: {
    x: number; // `x` coordinate inside the parent component
    y: number; // `y` coordinate inside the parent component
    width: number; // `width` of the TextInput
    height: number; // `height` of the TextInput
    absoluteX: number; // `x` coordinate on the screen
    absoluteY: number; // `y` coordinate on the screen
  };
};
```

Are you OK with this?

Reply:
@zoontek yes, I'm okay with that 🙂